feature-assets/src/main/java/io/novafoundation/nova/feature_assets/domain/assets/search/AssetSearchInteractor.kt

```diff
@@ -39,7 +40,9 @@ class AssetSearchInteractor(
         queryFlow: Flow<String>,
         externalBalancesFlow: Flow<List<ExternalBalance>>,
     ): Flow<Map<AssetGroup, List<AssetWithOffChainBalance>>> {
-        return searchAssetsInternalFlow(queryFlow, externalBalancesFlow) { asset ->
+        val comparator = compareByDescending(AssetGroup::groupTransferableBalanceFiat)
```

Comment:
We should also apply default chain sorting as a fallback `.then(Chain.defaultComparatorFrom(AssetGroup::chain))
`